runtime/string_functions.cpp

```diff
@@ -2327,6 +2327,10 @@ bool f$str_ends_with(const string &haystack, const string &needle) {
   return haystack.ends_with(needle);
 }
 
+bool f$str_contains(const string &haystack, const string &needle) {
+  return haystack.contains(needle);
```

Comment:
What are the advantages of implementing it as a new functions instead of using already existing strpos?

PHP uses similar approach:
https://github.com/php/php-src/blob/0b6f58d907e7169f7876bbea199967262197fbaa/ext/standard/string.c#L1794

As seen in the code below, str_contains could be implemented in terms of strpos like so:
```php
return '' === $needle || false !== strpos($haystack, $needle);
```

I would check what `php_memnstr` from PHP is. Maybe we have compatible function. If not, we could add one and use it in places where PHP uses it.

Maybe `php_memnstr` is identical to the newly introduced `contains` method, but I suggest re-checking that.